src/exampleMenu.ts

```diff
@@ -33,59 +33,17 @@ const molgenisMenu: MolgenisMenu = {
         id: 'navigator',
         label: 'Navigator'
       },
+      {
+        type: 'plugin',
+        id: 'navigator',
+        label: 'Navigator alt'
+      },
       {
         type: 'plugin',
         id: 'dataexplorer',
         label: 'Data Explorer',
         params: 'selectedEntityType=sys_md_EntityType'
       },
-      {
-        type: 'menu',
-        id: 'dataintegration',
-        label: 'Data Integration',
-        items: [
-          {
-            type: 'plugin',
-            id: 'metadata-manager',
-            label: 'Metadata Manager'
-          },
-          {
-            type: 'menu',
-            id: 'submenu',
-            label: 'Submenu',
-            items: [
-              {
-                type: 'plugin',
-                id: 'sorta',
-                label: 'SORTA'
-              },
-              {
-                type: 'menu',
-                id: 'subsubmenu',
-                label: 'Sub Sub Menu',
-                items: [
-                  {
-                    type: 'plugin',
-                    id: 'background',
-                    label: 'Test',
-                    'params': ''
-                  }
-                ]
-              },
-              {
-                type: 'plugin',
-                id: 'mappingservice',
-                label: 'Mapping Service'
-              }
-            ]
-          },
-          {
-            type: 'plugin',
-            id: 'tagwizard',
-            label: 'Tag Wizard'
-          }
-        ]
-      },
```

Comment:
I think it's better to delete the import plugin, as the menu now doesn't have submenu's anymore.

Reply:
done